eva-server/src/main/java/uk/ac/ebi/eva/server/ws/GeneWSServerV2.java

```diff
@@ -137,7 +137,14 @@ public ResponseEntity getVariantsByGene(
                 response, request), HttpStatus.OK);
     }
 
-    private void checkParameters(String species, String assembly, Integer bufferValue) throws IllegalArgumentException {
+    private void checkParameters(List<String> geneIds, String species, String assembly, Integer bufferValue)
+            throws IllegalArgumentException {
+        long emptyGeneIdCount = geneIds.stream().map(String::isEmpty).filter(value -> value == true).count();
```

Comment:
what this is checking is if any element is an empty string. I didn't think about this but it would be nice to search only non-empty strings, but my comment was that the endpoint allows to search by an empty *list*, which this method will allow right now.